backend/ops_api/tests/ops/features/edit_agreement_metadata.feature

```diff
@@ -22,3 +22,18 @@ Feature: Edit Agreement Metadata
     And I have a Contract Agreement
     When I submit a new value for notes
     Then I should get an message that it was successful
+
+  Scenario: Failed Edit because In Execution
+    Given I am a logged in as an OPS user
+    And I have a Contract Agreement with a BLI in execution
+    And I edit the agreement to change a value
+    When I submit the agreement
```

Comment:
I guess I'm questioning whether they should even be able to edit the agreement to begin with. i.e. - if the Agreement is in "Execution", then if you don't have permission (admin) should the edit fields be enabled?